hazelcast-client/src/main/java/com/hazelcast/client/util/AddressHelper.java

```diff
@@ -61,7 +61,7 @@ private AddressHelper() {
         try {
             inetAddress = InetAddress.getByName(scopedAddress);
         } catch (UnknownHostException ignored) {
-            Logger.getLogger(AddressHelper.class).finest("Address not available", ignored);
+            EmptyStatement.ignore(ignored);
```

Comment:
I am not sure about this one. it smells like a bad troubleshooting experience in a distance future :)